dj_secure_settings/loader.py

```diff
@@ -63,9 +63,9 @@ def load_secure_settings(project_name=None, environment=None):
         yaml_params = yaml.load(open(yaml_file), Loader=yaml.Loader)
         _load_params_from_yaml(config, yaml_params, env, 'defaults')
         _load_params_from_yaml(config, yaml_params, env, project_name)
-    except Exception as e:
+    except FileNotFoundError as e:
         # couldn't load params from a local file
-        logging.warning("Couldn't load params from local file: {}".format(str(e)))
+        logging.debug("Couldn't load params from local file: no local file exists")
```

Comment:
I recall that every time I get this wrong I have to go look in the code to figure out what the file should be, or troubleshoot why it's not finding the file I think it should -- could we update the message to include `yaml_file`? That might speed up troubleshooting when we have erroneous assumptions about those values. Also, maybe this message can say "File `yaml_file` not found" or something similar, to distinguish it better from the message on line 85?

Reply:
Sure -- I'll add that.